### ISSUE TITLE
rerdirect loop

### ISSUE THREAD

**Issue by bbunlock** [yii2mod/base#31]
Hi, first thanks for putting this together its saved me some time, I have installed the app and on my pc (xampp) and got it working perfectly, however im having problems on my live site, the problem is that when I login I am stuck in a redirect loop.

I have narrowed it down to when I have the remember me ticked on the login form, if it is not ticked the login works great and I am redirected to the home page, however this login only lasts for the session.

I have this on a shared hosting server so I have had to do a redirect from the root to the web folder and I think this may be the cause, I have tried many variations of the htaccess file in both the root and the web folder but still a redirect loop whenever I have the remember me ticked.

I have to delete the _identity cookie to be able to see the site again, I must point out that the redirect loop is on every page not just the index page.

If you have any ideas or suggestions on how to fix this I would appreciate it.

Kind regards

Wayne

**Comment by igor-chepurnoi** [comment]
Hi, please execute the following commands
```bash
$ php yii migrate
$ php yii rbac/migrate
```

**Comment by bbunlock** [comment]
no all the migrations were made, I have since tried a few other yii2 apps and none of them allow me to tick remember me so its not this app its yii2 and my server.

Thanks for a great app.

Kind regards

Wayne

**Comment by bbunlock** [comment]
just an update for you all that might be having his same problem, I recently updated my server software via cpanel, I updated the apache and also php, im not sure what version of php I was using before the update but as soon as I went to php 5.6 I was able to login and no redirect.

it would appear that yii2 apps or at least some require php 5.6 and higher.

once again thanks for the great work on the app.

Kind regards

Wayne
